src/lemlib/pose.cpp

```diff
@@ -11,6 +11,9 @@
 #include <math.h>
 #include "lemlib/pose.hpp"
 
+#define FMT_HEADER_ONLY
+#include "fmt/core.h"
```

Comment:
Clean up includes and defines
```suggestion
#define FMT_HEADER_ONLY

#include <math.h>

#include "fmt/core.h"
#include "lemlib/pose.hpp"
```